backend/api/Areas/Admin/Controllers/ParcelController.cs

```diff
@@ -233,15 +182,7 @@ public IActionResult AddParcels([FromBody] Model.ParcelModel[] models)
         [HttpPut]
         public IActionResult UpdateParcel([FromBody] Model.ParcelModel model)
```

Comment:
FYI I do not like how this controller is processing sub objects (buildings and addresses). There must be a nicer way to get EF to save those entities.

At the moment I'm leaving it because the scope of this store is already very large. We can fix it when we actually have an implementation story that uses this controller.

Reply:
I believe EF may add new sub objects correctly (not sure 100%), but I it won't correctly handles updates or deletes.  Additionally it won't handle more complicated relationships (i.e. many-to-many).  Outside of us manually doing it, we would have to create a smart parser that did it for us on any object.